### PR TITLE
Ignore '/' at url end

### PR DESCRIPTION
**Pull request by fabiuz7**
Hi, 
thanks for your extension. I was trying to use it but I found a couple of issues (described in the commits). 

Hope this helps

### CODE REVIEW

File: src/inject/index.js [hunk 1]
```diff
@@ -5,7 +5,10 @@
  * @returns {boolean}
  */
 function isLandPage() {
-  const uri = window.location.pathname.substring(1);
+  let uri = window.location.pathname.substring(1);
+  if(uri.slice(-1) === '/'){
+    uri = uri.slice(0, -1);  
+  }
```
Comment (lvarayut) [inline src/inject/index.js:11]: Looks good to me. It'd be awesome if the code is formatted

File: src/inject/index.js [hunk 2]
```diff
@@ -5,7 +5,10 @@
  * @returns {boolean}
  */
 function isLandPage() {
-  const uri = window.location.pathname.substring(1);
+  let uri = window.location.pathname.substring(1);
+  if(uri.slice(-1) === '/'){
```
Comment (lvarayut) [inline src/inject/index.js:9]: `uri.endsWith('/')` would be more readable. Wdyt?

File: src/inject/index.js [hunk 3]
```diff
@@ -119,7 +122,7 @@ function formatDate(date, format) {
  */
 function injectDateToHTML(date) {
   const aboutElement = document
-    .querySelector('.octicon-book')
+    .querySelector('.BorderGrid .octicon-book')
```
Comment (lvarayut) [inline src/inject/index.js:125]: The issue has already been fixed in c15008f48b69bde6124db778198e128aea8e2129

**Comment by lvarayut** [comment]
Hi Fabiano, thanks for your contribution. The changes look good to me. I have left a few minor comments.

**Comment by fabiuz7** [comment]
fix your comments, I'm not very fresh on JS, I used daily another languages :)

**Comment by lvarayut** [comment]
No worries :). Could you please revert the https://github.com/lvarayut/github-date-of-creation/pull/8/commits/9d7a73ba68c0d98a0a27517282902adeb3984d0a since it has been fixed in c15008f, so I can merge this PR?

**Comment by fabiuz7** [comment]
Done!

**Comment by lvarayut** [comment]
Awesome, thank you.

**Pull request merged**